manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "authors.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "authors.settings.development")
```

Comment:
Does this change the way django commands are run ? If yes, update the readme to reflect the same.

Reply:
I have included that in my PR readme

Reply:
@inno-asiimwe  no